Formula/twilio@rc.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class TwilioATrc < Formula
+  desc "unleash the power of Twilio from your command prompt"
+  homepage "https://github.com/twilio/twilio-cli"
+  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v2.27.0-rc.1/twilio-v2.27.0-rc.1.tar.gz"
+  version "2.27.0-rc.1"
+  sha256 "919ff4d3df952e61506f967dc63d0ad007a30a0ac0e78b22747a3aa93fd02139"
```

Comment:
[Q] How was this sha256 generated?

Reply:
It uses the existing logic. I haven't made any changes to that.